reviewing.md

```diff
@@ -10,10 +10,13 @@ Kubernetes for which conformance is being claimed, referenced as the
 3. Verify that the Conformance Version is the current or previous version of
 Kubernetes.
 
-4. Look at `version.txt`.  Verify that the `client` version matches the
-Conformance Version to a minor version level (the patch version may differ).
-`client` in this file represents the version of the e2e tests that were run.
+4. Look at `version.txt`.  Verify that the `major.minor` component of the 
+`server` version exactly matches the Conformance Version.
 
-5. Verify that the last line of `e2e.log` says "SUCCESS! -- `N` Passed | 0
+5. Look at `version.txt`.  Verify that the `major.minor` component of the 
+`client` version (which represents the Kubernetes version of the e2e tests that
+were run) is equal to or greater than the Conformance Version.
```

Comment:
remove "(which represents the Kubernetes version of the e2e tests that
 +were run)"

Reply:
The reason that text is there is that some people were confusing what "client" was, thinking it was `kubectl`. I'm open to rewording, but I think we need to provide *some* clarification.

Reply:
Is this `version.txt` file part of the output tarball? The correct way to create it doesn't seem to be covered in instructions.

Reply:
@dghubble yes, you must use the output from Sonobuoy which you should result from running the [instructions](https://github.com/cncf/k8s-conformance/blob/master/instructions.md).

Reply:
@WilliamDenniss I'm ok with it. 

LGTM